utilities/transactions/write_unprepared_txn.cc

```diff
@@ -729,7 +736,6 @@ Status WriteUnpreparedTxn::RollbackToSavePointInternal() {
   assert(flushed_save_points_->size() > 0);
   WriteUnpreparedTxn::SavePoint& top = flushed_save_points_->back();
 
-  assert(top.unprep_seqs_.size() > 0);
```

Comment:
Why is this assert removed?

Reply:
This is removed because now we may skip flushing a write batch, if that write batch is empty, but we still want to save the savepoint.

If we set savepoint right after begin transaction, then when we flush the write batch, the first savepoint would not have any unprep_seqs because no flush was done.